dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/relationship/RelationshipsExportControllerTest.java

```diff
@@ -478,6 +507,30 @@ void shouldGetRelationshipsByTrackedEntity() {
     assertHasOnlyUid(relationship1To.getUid(), "event", jsonRelationship.getTo());
   }
 
+  @Test
+  void
+      shouldGetNoRelationshipsByTrackedEntityWhenRelationshipTypeIsUnidirectionalAndTrackedEntityIsOnToEnd() {
+    RelationshipType relationshipType = manager.get(RelationshipType.class, "m1575931405");
+    relationshipType.setBidirectional(false);
+    manager.update(relationshipType);
+
+    assertEquals(
```

Comment:
Can you use the same setup like in

https://github.com/dhis2/dhis2-core/pull/20212/files#diff-ac504135039dad6a3a06c6969c94ff471fa3a1c9f1bd10f3441d0e48e2f872d4R354-R356

? Looks shorter. Or use the same setup for all the 3 tests you've added. The above 2 use yet another style to get the entity